exercises/saddle-points/uSaddlePointsTests.pas

```diff
@@ -1,3 +1,35 @@
+//*****************************************************
+// You got an error, which is exactly as it should be.
+// This is the first step in the Test-Driven Development
+// (TDD) process.
+//
+// The most important part of the error is
+//
+//   "cannot compile"
+//
+// It's looking for a file named uSaddlePoints.pas that doesn't
+// exist yet.
+//
+// To fix the error, create a unit file named uSaddlePoints.pas
+// in the same directory as the uSaddlePointsTests.pas file.
+//
+// The beginning of the new unit file should contain a unit statement:
+// unit uSaddlePoints;
+//
+// The new unit should contain Interface, Implementation, and End. statements.
+//
+// Hint: Delphi will take care of all this if you instruct it to add a new unit
+// to your project.  Be sure to save the new unit as uSaddlePoints.pas before
+// trying to compile again.
+//
+// Hint: You will be using TTuple in your solution.  TTuple is
```

Comment:
From lines 1 to line 24, I think that unless this is the 2nd or maybe 3rd time, the hand holding can be here, but at some point, this information should be inferred by the compile and test failure messages.

The information about the TTuple is the "new information" and so it is good here.